content/blog/building-original-projects/index.md

```diff
@@ -0,0 +1,285 @@
+# Make a plan to solve your problem
+Now that you have the destination, you need a route to get there. A plan is your road map to success that will guide you through confusion and keep you on the right track to your original goals. If you try to proceed without a written plan, you might get lost in the details and lose track of the bigger picture. You can keep your plan in a physical or digital format, whichever you prefer.
+## Brainstorm
+I recommend starting with a brainstorming session where you jot all of your ideas and thoughts down somewhere. Take this as an opportunity to think through your approach and what exactly you want your program to accomplish. You can also decide what programming languages, frameworks or libraries you might want to use here.
+
+For example, here is a fake brainstorming session for a hypothetical website: *I want to create an indie game blog site that let's people create accounts and write reviews of their favorite indie games, share their hours played/achievements from their Steam account at the top of the review, and upload video files to insert into the blogpost for an audio-textual experience that clearly describes the game. At the end of the blogpost, links with image preview thumbnails to Twitch streams of the game being reviewed are provided for the reader in case they want to see the game in action. These could be current livestreams of the game or past livestreams of the writer.*
+## Step-by-step plan
+Next, I recommend taking that unorganized jumble of ideas you've produced and unpacking them into a neater, but still rough sketch of your project that outlines that core features at a high level. If you've generated tons of ideas, it's also important to start separating core, essential features from nice-to-have-but-unnecessary-features. For our indie game blog, the core functionality boils down to:
+
+- Authorization: You need a way for users to sign up, log in and log out.
+- Blogposts: You need a way for writers to create and save new blogposts, embed videos and pictures.
+- Displaying data: Users visiting the site should be able to see recently posted articles upon landing on your site.
+- API access: You need to grab data from Steam and Twitch, and display it on the page.
+- User interface: How should the layout of the website look visually? You might need some mockups. 
+
+Each of these can be further broken down into subtasks, until you end up with something like:
+-  Implementing Blogposts
+	1. First create simple text-only posts that can be posted by authorized users and publicly viewed by anyone. Make some pages with forms for creating new posts.
+	2. Look into how to allow authors to upload/insert video files to their blog post.
+	3. Find a way to link Steam account data to author account, and display relevant information such as the amount of hours the author has played this game.
+	4. With Twitch API access in hand, display past streams of the game at the bottom of any given blogpost.
```

Comment:
While I love the idea of a list here - as it stands it feels incomplete:

<img width="482" alt="image" src="https://github.com/unicorn-utterances/unicorn-utterances/assets/9100169/acb3c746-178f-4f90-8d58-6ff28e01815a">

To fix this, I would:

- Add more root (non-numbered) bullet-points
- Make the list match the `Authorization` and `Blog posts` lists from above. Parity is key here for glanceability
- Don't be afraid of nested list items
  > Something like this nested block quote might help add visual diversity

Reply:
I updated the section, was that closer to what you had in mind?

Reply:
LGTM!